libnetwork_test.go

```diff
@@ -1132,6 +1119,102 @@ func TestResolvConf(t *testing.T) {
 	}
 }
 
+func TestInvalidRemoteDriver(t *testing.T) {
+	if !netutils.IsRunningInContainer() {
+		t.Skip("Skipping test when not running inside a Container")
+	}
+
+	mux := http.NewServeMux()
+	server := httptest.NewServer(mux)
+	if server == nil {
+		t.Fatal("Failed to start a HTTP Server")
+	}
+	defer server.Close()
+
+	type pluginRequest struct {
+		name string
+	}
+
+	mux.HandleFunc("/Plugin.Activate", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "appplication/vnd.docker.plugins.v1+json")
```

Comment:
Thanks. Its a copy+paste issue originating from the plugins code :-(
This needs a fix in the plugins pkg (which we have imported as dependency).

I will first fix the Docker pkg, import it and then change it here to be consistent.